src/services/wallet.singleton.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Some suggestions for this file.

Maybe rename `getGlobalWallet` to `useGlobalWallet`.
rename `resetWallet` to `stopWallet`.

The file itself, instead of `src/services/wallet.singleton.js` why not `src/modules/wallet.js`?
Since this is not really a service or an utility and naming it state or store would conflict with redux I think modules is a better alternative.
We could also name it `src/hooks/wallet.js` (since it's functionality is basically a custom hook).
What do you think?

Reply:
Indeed, this singleton is in a kind of a limbo, because it represents an object that is actually a big compilation of helpers, services and user data by itself: the `HathorWallet` object.

I went for `modules/wallet.js`, as it felt more adequate to this description above.

About the `get` or `use` prefix, I followed the [naming convention](https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use) for React functions that states that a hook function should call one or more hooks inside it. That way, linters and other static code tools can correctly identify this function belongs to a React environment.

Since the `globalWallet` retrieval function does not use hooks and probably won't in the future, it seems more correct to keep it as a `getGlobalWallet`.